composer.json

```diff
@@ -16,10 +16,10 @@
     },
     "license": "GPL-3.0+",
     "require": {
-        "php": ">=5.4.0",
-        "ext-bcmath": "*"
+        "php": ">=5.4.0"
     },
     "require-dev": {
+        "ext-bcmath":                               "*",
```

Comment:
If you are not agree with this, so we have mad a misunderstanding and this PR should be closed.